spec/features/course/assessment/assessment_attempt_spec.rb

```diff
@@ -93,6 +103,49 @@
         end
       end
 
+      scenario 'I can view tabbed assessments and tabs for assessments with more than 1 question',
+               js: true do
+        assessment_tabbed_single_question
+        visit course_assessments_path(course)
+
+        within find(content_tag_selector(assessment_tabbed_single_question)) do
+          find_link(
+            I18n.t('course.assessment.assessments.assessment_management_buttons.attempt'),
+            href: course_assessment_submissions_path(course, assessment_tabbed_single_question)
+          ).trigger('click')
+        end
+
+        expect(page).not_to have_selector('ul.nav.nav-tabs.tab-header')
+
+        assessment_tabbed
+        visit course_assessments_path(course)
+
+        within find(content_tag_selector(assessment_tabbed)) do
+          find_link(
+            I18n.t('course.assessment.assessments.assessment_management_buttons.attempt'),
+            href: course_assessment_submissions_path(course, assessment_tabbed)
+          ).trigger('click')
+        end
+
+        expect(page).to have_selector('ul.nav.nav-tabs.tab-header')
+
+        within find('.tab-pane.active') do
+          # Test that the first tab is loaded, and that ACE Editor has initialised correctly.
+          expect(page).to have_selector('div.ace_editor')
+        end
+
+        # Click on tab of second question
+        question_id = assessment_tabbed.questions.second.id
+        find(".tab-header a[href='##{question_id}']").click
+        sleep 0.2
```

Comment:
@allenwq I needed to add this in because the `.fade` class when changing tabs introduces delay. Without the sleep, the next assertion does not hold true (because the tab-content has not changed yet). 

This `sleep 0.2` is not ideal, I will fix this after it is merged. One possible solution is to remove all `.fade` class, but there are other methods too. 

Got the solution from here -> http://stackoverflow.com/questions/14488836/trouble-interacting-with-bootstrap-modals-via-capybara-v2